eng/Version.Details.xml

```diff
@@ -14,7 +14,7 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>61cde6e8fb9d5c9790867b279deb41783a780cd8</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Helix.Sdk" Version="5.0.0-beta.20502.2">
+    <Dependency Name="Microsoft.DotNet.Helix.Sdk" Version="5.0.0-beta.20502.7">
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>baebe30c7969b4148a6a3b90cf5f4e3a96d4a11e</Sha>
```

Comment:
Technically this value should be `b931c53efeaac8270b319defc0905e9c152bac1a` (guessing you did this manually) but it may not matter

Reply:
Yeah I never update the commit SHA when I do these kinds of manual updates and it doesn't seem to be a problem.